src/main/resources/templates/db_scripts/migratePrecedingSucceedingTitles.sql

```diff
@@ -0,0 +1,22 @@
+START TRANSACTION;
+
+INSERT INTO ${myuniversity}_${mymodule}.preceding_succeeding_title (id, jsonb,
+	   creation_date, created_by, precedingInstanceId, succeedingInstanceId)
+  SELECT id, jsonb_build_object(
+    'id', jsonb->'id',
+	'metadata', jsonb_build_object(
+		'createdDate', jsonb->'metadata'->'createdDate',
+		'updatedDate', jsonb->'metadata'->'updatedDate'),
```

Comment:
Can it be simplified to?

```suggestion
	'metadata', jsonb->'metadata'
```